iceoryx_posh/include/iceoryx_posh/internal/popo/ports/client_port_user.hpp

```diff
@@ -31,6 +31,13 @@ namespace iox
 {
 namespace popo
 {
+enum class ClientSendError
+{
+    NO_CONNECT_REQUESTED = 2, // start at 2 to make the C binding match
```

Comment:
In the C-binding there is a `SUCCESS` value and an `UNDEFINED_ERROR`, the first one is `0`, the second one `1`. When this enum starts at 2, the remaining underlying values are equal. I thought it would be a good idea but can also revert this